### PR TITLE
Add revision tag documentation to canary upgrade docs

### PR DESCRIPTION
**Pull request by Monkeyanator**
Document how to use revision tags to get stable namespace labels.

[ ] Configuration Infrastructure
[ ] Docs
[x] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/upgrade/canary/index.md
Comment (zerobfd) [inline content/en/docs/setup/upgrade/canary/index.md:116]: Suggestion:

```
Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are a solution to this. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are stable identifiers that point to revisions and can be used to avoid relabeling namespaces. Rather than relabeling the namespace, a mesh operator can simply change the tag to point to a new revision. All namespaces with that tag will be updated at the same time.
```

File: content/en/docs/setup/upgrade/canary/index.md
Comment (zerobfd) [inline content/en/docs/setup/upgrade/canary/index.md:118]: Suggestion: add
`This state could be reached via these commands:`

**Review by zerobfd — APPROVED** [review]
I have a few wording suggestions, but overall lgtm.

**Comment by Monkeyanator** [comment]
Thanks @zerobfd, incorporated the feedback

File: content/en/docs/setup/upgrade/canary/index.md
Comment (ericvn) [inline content/en/docs/setup/upgrade/canary/index.md:143]: Do you mean `A` and `B`? `C` was always on 1.8.0.

Comment (Monkeyanator) [reply content/en/docs/setup/upgrade/canary/index.md]: Fixed, thanks for catching

**Comment by ericvn** [comment]
@Monkeyanator The `doc.test.multicluster_istio.io` (external control plane) failure is currently expected.

**Comment by ericvn** [comment]
@istio/wg-environments-maintainers Any issues with this going in?

**Comment by ostromart** [comment]
lgtm

**Review by ericvn — APPROVED** [review]

**Review by morvencao — APPROVED** [review]

**Pull request merged**
